cwms-data-api/src/main/java/cwms/cda/data/dto/timeseriesprofile/TimeValuePair.java

```diff
@@ -0,0 +1,55 @@
+package cwms.cda.data.dto.timeseriesprofile;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+
+import java.time.Instant;
+
+@FormattableWith(contentType = Formats.JSONV2, formatter = JsonV2.class)
```

Comment:
Is this ever formatted on it's own or always as part of some container object? If always within a container it doesn't need the FormattableWith annotation.

Reply:
yes it is always part of a container

Reply:
removed the annotation